apps/playground-web/src/components/styled-connect-button.tsx

```diff
@@ -17,6 +18,7 @@ export function StyledConnectButton(
       supportedNFTs={{
         "84532": ["0x638263e3eAa3917a53630e61B1fBa685308024fa"],
       }}
+      chain={sepolia}
```

Comment:
remove this, fixed default chains in my PR, this actually forces sepolia which we dont want